client/src/components/Pill/Pill.css

```diff
@@ -0,0 +1,16 @@
+.pill-container {
+  height: 1.061vw;
+  width: 7.071vw;
+  position: absolute;
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  font-family: non-serif;
+  bottom: 0%;
+  font-size: 0.707vw;
+  font-family: sans-serif;
+  font-weight: 350;
+  text-align: center;
+  color: white;
```

Comment:
Make sure that you are using the CSS variables when referring to colors. 
So, here it would be color: var(--white); 
The color isn't actually pure white.